suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -152,6 +152,19 @@ export const selectAccountsByNetworkSymbols = memoizeWithArgs(
     },
 );
 
+export const selectAccountsByNetworkSymbol = memoizeWithArgs(
+    (state: AccountsRootState, networkSymbol: NetworkSymbol | null) => {
+        if (G.isNull(networkSymbol)) return [];
+
+        const accounts = selectAccounts(state);
+
+        return A.filter(accounts, account => account.symbol === networkSymbol);
+    },
+    {
+        size: 16,
```

Comment:
Why 16? If that is number of supported networks you can use something like `NetworksConfig.length` so we don't need to update this in future

Reply:
Exact networks config length added in 45943ed69